src/components/PupilComponents/QuizRenderer/QuizRenderer.tsx

```diff
@@ -87,78 +72,29 @@ export const QuizRenderer = () => {
     };
 
     innerRender = (
-      <OakForm onSubmit={handleSubmit}>
+      <OakForm id="a-form" onSubmit={handleSubmit}>
```

Comment:
It troubles me that the encapsulation is now broken for this component and  PupilQuizView. Can you turn this id into a prop and have pupilQuizView pass it ?

Reply:
good idea